src/app/reporting/Engine.h

```diff
@@ -119,6 +127,8 @@ class Engine
      *
      */
     uint32_t mCurReadHandlerIdx = 0;
+
+    ClusterInfo * mpChangePathList = nullptr;
```

Comment:
`ClusterInfo` unfortunately is not a very good name for this type. It really should be an `AttributePath` (to match the terminology in the spec).

Reply:
I would follow-up it in https://github.com/project-chip/connectedhomeip/issues/9552